pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+# Optimism
+-----------
+
+import Markdown from 'markdown-to-jsx'
+import Tabs from '../../../components/tabs'
+import CodeBlock from '../../../components/code-block'
+import { NextraTab, NextraTabs } from 'nextra-theme-docs'
+
+Instructions to set up your Optimism node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 80GB+ free storage space.
+- MacOS or Ubuntu 20.04 (tested on 20.04)
+- [Official Documentation](https://community.optimism.io/docs/developers/build/run-a-node/#)
+
+
+## Prerequisites
+```bash
+sudo apt-get install jq -y
+snap install docker 
+```
+
+## Clone Repo
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/smartcontracts/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/ethereum-optimism/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+}
+]} />
+
+
+## Set variables
```

Comment:
Can you clarify where these should be set? What's the best practice here? These have to be loaded into the user's shell for the following commands to work.

Reply:
Yes, we just need to set these in shell